src/Common/src/System/Data/Common/AdapterUtil.Drivers.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace System.Data.Common
+{
+    internal static partial class ADP
+    {
+
+        internal static Timer CreateGlobalTimer(TimerCallback callback, object state, int dueTime, int period)
```

Comment:
Nit: I would call it UnsafeCreateTimer.  Then it can be replaced by Timer.UnsafeCreate if/when it's added.

Reply:
Changed